src/main/java/browserview/BrowserComponent.java

```diff
@@ -537,35 +568,32 @@ public String getCurrentUrl() {
     /**
      * Switches to the specified tab in GitHub PR page
      *
-     * @param tabName Either GithubPageElements.DISCUSSION_TAB, GithubPageElements.COMMITS_TAB
-     *                or GithubPageElements.FILES_TAB
+     * @param pr
```

Comment:
- Give some explanation about `pr`.
- Update the main comment to `Opens a specified tab of a PR` or something similar since we are no longer switching.
- Keep `tabName` verbose to the enums, since we have not really separate them into their own enum.